Add glitch practice cheats

Pepe20129 opened the pull request:
Adds a cheat to prevent fish from despawning, a cheat to prevent bugs from despawning and a cheat to change the fuse timer of bombs.
Meant to help with practicing glitches.

<!--- section:artifacts:start -->
### Build Artifacts
  - [soh.otr.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/850557247.zip)
  - [soh-linux-compatibility.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/850557248.zip)
  - [soh-linux-performance.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/850557250.zip)
  - [soh-mac.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/850557251.zip)
  - [soh-switch.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/850557253.zip)
  - [soh-wiiu.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/850557254.zip)
  - [soh-windows.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/850557255.zip)
<!--- section:artifacts:end -->

Pepe20129 committed:
Add fish don't despawn cheat

Pepe20129 committed:
Add bugs don't despawn cheat

Pepe20129 committed:
Add bomb timer multiplier cheat

Pepe20129 committed:
Only do float logic when multiplier isn't 1

briaguya0 commented:
this isn't a bugfix so it should be pointed to develop, not khan

garrettjoecox reviewed (approved):
LGTM, just needs a rebase

Edit: Also can you add the new cvars to the presets.h under enhancmentCvars

Pepe20129 committed:
Move don't despawn cheats and fix capitalization

Pepe20129 committed:
Fix LUS commit change

Pepe20129 committed:
Re-add the bomb timer multiplier slider

Pepe20129 committed:
Add cvars to cheatCvars

Pepe20129 commented:
I think I rebased correctly but it's my first time rebasing a pr so I'm not sure.
I added the cvars to `cheatCvars` instead of `enhancementsCvars`.
